### ISSUE TITLE
Exception thrown in Stories view

### ISSUE THREAD

**Issue by beevelop** [botfront/botfront#242]
With the current version of Botfront, I am experiencing a reproducible exception when trying to view the `stories` view. The full exception is:
```
TypeError: Cannot read property 'forEach' of undefined
    at d (a598bcc16375ac00b89878e27278ded85aaaed27.js?meteor_js_resource=true:149)
    at k (a598bcc16375ac00b89878e27278ded85aaaed27.js?meteor_js_resource=true:149)
    at eval (/imports/ui/components/stories/StoryEditorContainer.jsx:1)
    at Ta (a598bcc16375ac00b89878e27278ded85aaaed27.js?meteor_js_resource=true:9)
    [...]
```

The issue happened in my case, when upgrading an existing Botfront instance from `0.15.4` to anything "above it" (means starting from 0.15.5). There is no exception thrown on the server-side. I think the issue is caused by the mentioned `StoryEditorContainer` component: Maybe https://github.com/botfront/botfront/blob/master/botfront/imports/ui/components/stories/StoryEditorContainer.jsx#L135

I experimented with multiple configurations, e.g.:
```
BOTFRONT_VERSION=v0.16.0-rc.10
BOTFRONT_API_VERSION=v0.16.1
RASA_FOR_BOTFRONT_VERSION=v1.3.7-bf.1
```

**Comment by MatthieuJnon** [comment]
This has been fixed in the latest version v0.16.0-rc.12, please update to that one. Sorry for the trouble

**Comment by beevelop** [comment]
Wow, that was quick. Thank you for the fix.
